package.json

```diff
@@ -3,7 +3,7 @@
 	"displayName": "MetaEd IDE",
 	"description": "Ed-Fi MetaEd IDE for Visual Studio Code",
 	"license": "SEE LICENSE IN LICENSE.md",
-	"version": "4.2.1",
+	"version": "4.2.2-dev.22",
```

Comment:
The latest one in Azure Artifacts is `dev.21`.

Reply:
I pushed before I was done committing 🙄